### PR TITLE
Add meta links, tags and microdata, related to SEO, issue #406…

### PR DESCRIPTION
**Pull request by sneetsher**
… SEO issue

* Add canonical link to HTML header with only 'query' as parameter
* Remove the tailing '?' from alternative link in HTML header for empty requests

### CODE REVIEW

**Comment by assem-ch** [comment]
is there a way to deal with page ?

**Comment by sneetsher** [comment]
@assem-ch Yes, as I remember, our parameters for `page` is generic and no need to add more tags. Let me, check the reference.

**Comment by assem-ch** [comment]
@sneetsher okay let me know when you fix it

**Comment by sneetsher** [comment]
Okey @assem-ch , I'm fixed pagination rel-links .

File: src/alfanous-django/wui/templates/wui/result/header.html [hunk 1]
```diff
@@ -1,5 +1,6 @@
 {% load query %}
 
+<h1>
```
Comment (assem-ch) [inline src/alfanous-django/wui/templates/wui/result/header.html]: is that the page header or aya header?

Comment (assem-ch) [reply src/alfanous-django/wui/templates/wui/result/header.html]: yes only one h1 for the page , an i think h2 for query section, keywords, suggestions, aya results ... and h3 for each aya result

File: src/alfanous-django/templates/aya_search.html [hunk 2]
```diff
@@ -14,16 +14,16 @@
 <div class="control-group">
   <div class="controls">
     <label class="checkbox inline">
-    	<input type="checkbox"  name="fuzzy" value="True" {{ params.fuzzy|yesno:"checked," }} /> 
-		{{ _("Fuzzy search") }}
+      <input type="checkbox"  name="fuzzy" value="True" {% if params.query  %}{{ params.fuzzy|yesno:"checked," }}{% else %}checked="yes"{% endif %} />
```
Comment (assem-ch) [inline src/alfanous-django/templates/aya_search.html]: what about rename it exact search, and reverse the logic

Comment (sneetsher) [reply src/alfanous-django/templates/aya_search.html]: Yeah, 'Exact' is well known for most users.

Comment (sneetsher) [reply src/alfanous-django/templates/aya_search.html]: left it as todo, it need logic should be inverted in js. and it requires translation update.

File: src/alfanous-django/templates/base.html [hunk 3]
```diff
@@ -281,16 +281,18 @@
...
+            <main>
             <div class="row offset2 span9 text-center">
                 <div class="span7 offset1">
+                  <h1>
                     <a class="logo" href="{% url 'wui.views.results' current.unit %}">
                     {% if LANGUAGE_CODE == "ar" or  LANGUAGE_CODE == "fr" or LANGUAGE_CODE == "id" %}
-                  	  <img src="/static/shared/img/logo_{{ current.unit }}_{{ LANGUAGE_CODE }}.png" alt="logo">
-                  	 {% else %}
-                  	 	 <img src="/static/shared/img/logo_{{ current.unit }}_en.png" alt="logo">
-                  	  {% endif %}
```
Comment (assem-ch) [inline src/alfanous-django/templates/base.html]: I think H1 should have a unique value by page, the query for example

Comment (sneetsher) [reply src/alfanous-django/templates/base.html]: Most search pages I looked at in the web uses fix one. Could we recheck that after trying that with google. and see reports.

Comment (assem-ch) [reply src/alfanous-django/templates/base.html]: @sneetsher no problem, is the PR complete now and ready for merge?

Comment (sneetsher) [reply src/alfanous-django/templates/base.html]: just one minor update for sitemap to fix google warning about it's spec. We keep microdata for another separate PR.

**Comment by sneetsher** [comment]
Ok @assem-ch I'm done here .

**Pull request merged**